packages/adapter/addon/-private/utils/serialize-into-hash.js

```diff
@@ -0,0 +1,11 @@
+export default function serializeIntoHash(store, modelClass, snapshot, options = { includeId: true }) {
+  const serializer = store.serializerFor(modelClass.modelName);
+
+  if (typeof serializer.serializeIntoHash === 'function') {
```

Comment:
We don't expect this to be present? The current code that this is replacing (e.g. in packages/adapter/addon/json-api) didn't guard...

Reply:
We didn’t mean for it to be required, and if it’s not present we can safely do a different thing. What this PR does (in addition to the new docs) is ensure that things we hadn’t intended to be required in the past are truly optional.